openpype/plugins/publish/integrate_new.py

```diff
@@ -913,21 +652,19 @@ def create_version_data(self, context, instance):
             source = context.data["currentFile"]
             anatomy = instance.context.data["anatomy"]
             source = self.get_rootless_path(anatomy, source)
-
         self.log.debug("Source: {}".format(source))
+
         version_data = {
-            "families": families,
+            "families": get_instance_families(instance),
             "time": context.data["time"],
             "author": context.data["user"],
             "source": source,
             "comment": context.data.get("comment"),
             "machine": context.data.get("machine"),
-            "fps": context.data.get(
-                "fps", instance.data.get("fps")
-            )
+            "fps": instance.data.get("fps", context.data.get("fps"))
```

Comment:
Having `instance.data["fps"]` override `context.data["fps"]` if it exists made more sense to me. Is that correct thinking?

Reply:
I agree